silverback/_cli.py

```diff
@@ -221,7 +221,10 @@ def workspace_info(platform: "PlatformClient", workspace: str):
 
     click.echo(f"{click.style('Name', fg='green')}: {workspace_info.name}")
     click.echo(f"{click.style('Slug', fg='green')}: '{workspace_info.slug}'")
-    click.echo(f"{click.style('Date Created', fg='green')}: '{workspace_info.created}'")
+    click.echo(
+        f"{click.style('Date Created', fg='green')}: "
+        f"{workspace_info.created.astimezone().strftime('%Y-%m-%d %H:%M:%S %Z')}"
```

Comment:
how does it display differently if you don't have `.strftime('%Y-%m-%d %H:%M:%S %Z')` to render it?

Reply:
Without
```
Date Created: 2024-11-21 15:17:44.504808-06:00
```
With
```
Date Created: 2024-11-21 15:17:44 CST
```

Reply:
Okay, that looks good. Perhaps you might want to make the format string a constant that you can re-use in different places here